Build failing:

This one looks related to the change from `RegexURLResolver => URLResolver` 
```
ERROR: corehq.apps.app_manager.tests.test_views:TestViews.test_download_file_bad_xform_404
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python3.6/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_views.py", line 96, in test_download_file_bad_xform_404
    path='modules-0/forms-0.xml')))
  File "/vendor/lib/python3.6/site-packages/django/test/client.py", line 536, in get
    **extra)
  File "/vendor/lib/python3.6/site-packages/django/test/client.py", line 340, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/vendor/lib/python3.6/site-packages/django/test/client.py", line 416, in generic
    return self.request(**r)
  File "/vendor/lib/python3.6/site-packages/django/test/client.py", line 501, in request
    six.reraise(*exc_info)
  File "/vendor/lib/python3.6/site-packages/django/utils/six.py", line 686, in reraise
    raise value
  File "/vendor/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/vendor/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/vendor/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/decorators.py", line 121, in _safe_cached_download
    response = f(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/views/download.py", line 349, in download_file
    return callback(request, domain, app_id, *callback_args, **callback_kwargs)
TypeError: 'str' object is not callable
```

--- a/corehq/apps/app_manager/views/download.py
+++ b/corehq/apps/app_manager/views/download.py
@@ -7,7 +7,12 @@ from django.contrib import messages
 from django.http import Http404, HttpResponse, JsonResponse
 from django.shortcuts import render
 from django.template.loader import render_to_string
-from django.urls import RegexURLResolver, Resolver404
+try:
+    from django.urls import URLResolver
+except ImportError:
+    from django.urls import RegexURLResolver as URLResolver
+
+from django.urls import Resolver404
 from django.utils.translation import ugettext_lazy as _
 
 from couchdbkit import ResourceConflict, ResourceNotFound
@@ -261,7 +266,7 @@ def download_file(request, domain, app_id, path):
         full_path = 'files/%s' % path
 
     def resolve_path(path):
-        return RegexURLResolver(
+        return URLResolver(
             r'^', 'corehq.apps.app_manager.download_urls').resolve(path)
 
     def create_build_files(build_profile_id=None):

--- a/settings.py
+++ b/settings.py
@@ -1045,7 +1045,6 @@ TEMPLATES = [
             'loaders': [
                 'django.template.loaders.filesystem.Loader',
                 'django.template.loaders.app_directories.Loader',
-                'django.template.loaders.eggs.Loader',
             ],
         },
     },

